Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3-slim
+
+# For healthcheck
+RUN apt-get update && apt-get install curl -y
+
+# Install python requirements
+COPY requirements.txt /tmp/
+RUN pip install --upgrade pip && \
+    pip install --no-cache-dir -r /tmp/requirements.txt
+
+# Copy server files
+COPY . /usr/src/app/.
```

Comment:
Note: This copies everything over from the repo directory. Not optimal, but works for now. Would like to revisit this at some point to only copy necessary files.